at-hoangtran/Week3/Day4/Exercise3-Day4.rb

```diff
@@ -0,0 +1,44 @@
+class HtmlToText
+  def get_html(html)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/b841bab6cd381fd2aea25772a4ea14c33b24929d/at-hoangtran/Week3/Day4/Exercise3-Day4.rb#L2"></span>Metrics/PerceivedComplexity: Perceived complexity for get_html is too high. [10/8]</td>
    </tr>
  </tbody>
</table>